code/_onclick/hud/swarmer.dm

```diff
@@ -1,68 +1,75 @@
 /atom/movable/screen/swarmer
 	icon = 'icons/hud/swarmer.dmi'
 
+/atom/movable/screen/swarmer/MouseEntered(location, control, params)
+	. = ..()
+	openToolTip(usr, src, params, title = name, content = desc)
+
+/atom/movable/screen/swarmer/MouseExited(location, control, params)
+	closeToolTip(usr)
```

Comment:
Is there a reason these aren't action buttons? This seems like a lot of the same behavior from that bit of code

Reply:
You want me to convert all swarmer HUD buttons to action buttons? I do not think that would really improve usability as now they are more intuitively spread around instead of being in one line, so I would rather not do that. I derived this code from Blob HUD, which also uses this way for showing tooltips.

Reply:
I don't want that, I wanted to know why they weren't action buttons, since this seemed like a lot of the same logic.

Your answer is disappointing because it hurts inside, but it's understandable. Thank you